portfolio2023/aboutme.html

```diff
@@ -0,0 +1,34 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Yolis' Portfolio</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+    <body>
+        <div class="navbar">
+            <ul>
+                <li><a href="#">Home</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </div>
+        <img id="selfie" src="images/techtonicapic.jpeg" width="450px">
+             <div class="bioparagraph">
+                <p>
+                <h1 class="heading">Just A Latina Breaking Into Tech</h1>
```

Comment:
```suggestion
                <h1 class="heading">Just A Latina Breaking Into Tech</h1>
                <p>
```

A `<p>` does not allow for any [block-level tags](https://developer.mozilla.org/en-US/docs/Web/HTML/Block-level_elements) inside of it; the `<h1>` tag is a block-level element so it should not live inside a `<p>`